src/planner_test.cc

```diff
@@ -17,6 +16,28 @@
 // TODO: This should work being set to 1
 #define SPEED_STEP_FACTOR 4
 
+// Helper function for implementing ASSERT_NEAR.
+testing::AssertionResult DoubleNearAbsRelPredFormat(const char *expr1,
+                                                    const char *expr2,
+                                                    const char *rel_error_expr,
+                                                    double val1, double val2,
+                                                    double rel_error) {
+  const double diff = fabs(val1 - val2);
+  const double fmax = std::max(fabs(val1), fabs(val2));
+  if (diff <= rel_error * fmax) return testing::AssertionSuccess();
+
+  return testing::AssertionFailure()
+         << "The difference between " << expr1 << " and " << expr2 << " is "
+         << diff << ", which exceeds " << rel_error_expr << " * max(abs("
+         << expr1 << "), abs(" << expr2 << ")), where\n"
+         << expr1 << " evaluates to " << val1 << ",\n"
+         << expr2 << " evaluates to " << val2 << ",\n"
+         << rel_error_expr << " evaluates to " << rel_error << ".";
+}
+
+#define EXPECT_NEAR_REL(val1, val2, abs_error, rel_error) \
```

Comment:
You don't use `abs_error` anymore, so that parameter should go :)

Reply:
(solved)